dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -46,9 +59,11 @@ public static Builder builder() {
   public static DynatraceMetricExporter getDefault() {
     Builder builder = new Builder();
     try {
-      builder.setUrl(new URL("http://127.0.0.1:14499/metrics/ingest")).build();
+      builder
+          .setUrl(new URL("http://127.0.0.1:14499/metrics/ingest"))
+          .setEnrichWithOneAgentMetaData(true);
     } catch (Exception e) {
```

Comment:
```suggestion
    } catch (MalformedURLException e) {
```
Should be sufficient, right?